.gitignore

```diff
@@ -35,8 +34,9 @@ scipts/docker/nginx-conf/*
 
 # App
 build
-public/lib
```

Comment:
Should we add this line back to ensure folks have a smooth transition?

This folder will exist with 1500 files and they'll appear in their `git status` the moment this goes to `master`.